float64[6]  jaco_joint          # The six Jaco joints
float64     preshape_joint      # The spread joint for the Reflex fingers
float64[3]  proximal_joint      # The proximal joint for the Reflex fingers
float64[3]  distal_joint        # The distal joint for the Reflex fingers